src/test/ui/parser/doc-comment-in-stmt.stderr

```diff
@@ -19,6 +24,11 @@ LL |     false
 LL |     /*! bar */
    |     ^^^^^^^^^^ unexpected token
    |
+help: Consider removing this token
```

Comment:
It is somewhat misleading to call this a "token"

Reply:
What name would you suggest, i am willing to adjust this

Reply:
> Can you special-case this for certain tokens?
> 
> Also, not sure if the suggestion to remove a comment is useful here, since comments are usually intentionally placed. This one might be better fixed with a separate suggestion.

Oh, yea i can special case different token kinds

Reply:
Okay, so i made it vary based on the kind of token